subworkflows/local/alevin.nf

```diff
@@ -31,57 +31,49 @@ workflow SCRNASEQ_ALEVIN {
         """Must provide a genome fasta file ('--genome_fasta') and a gtf file ('--gtf'), or a genome fasta file
         and a transcriptome fasta file ('--transcript_fasta`) if no index is given!""".stripIndent()
 
-    assert txp2gene || gtf:
-        "Must provide either a GTF file ('--gtf') or kallisto gene map ('--kallisto_gene_map') to align with kallisto bustools!"
+    if (transcript_fasta) {
+        assert txp2gene:
+            "Since a built transcript was provided ('--transcript_fasta'), must also provide a simpleaf gene map ('--txp2gene') to use with simpleaf quant!"
```

Comment:
You probably meant to remove the check for salmon above rather than the one for Kallisto? 

Btw, can kallisto and salmon use the same parameter for the gene map? i.e. is `--kallisto_gene_map` and `--txp2gene` the same thing?

Reply:
Actually, I thought on removing and adapting the kallisto one because the alevin workflow does not actually use kallisto bustools, so the asserting was not actually right.

But, I may be wrong on that as well. How do you think the asserting should look like?

And I am not sure about the second question, if they are indeed the same. But, if they are serving the same purpose, I think we can maybe change it to a single parameter, e.g., `--txp2gene`.

Reply:
Sorry I missed that this is all within the aleving subworkflow, I thought it was in the global workflow. 

In that case you are completely right and it should never have been there. 

> But, if they are serving the same purpose, I think we can maybe change it to a single parameter

agree, but I have no idea. Asked on slack, but that can be addressed in a separate PR if necessary.